packages/lib/src/vis/shared/SelectionMesh.tsx

```diff
@@ -51,6 +66,9 @@ function SelectionMesh(props: Props) {
 
   const onPointerUp = useCallback(
     (evt: ThreeEvent<PointerEvent>) => {
+      if (!isDragging) {
+        return;
+      }
```

Comment:
This prevents the execution of the callback if `onPointerDown` a valid starting point was not found